packages/blade/src/components/Card/_decisions/decisions.md

```diff
@@ -0,0 +1,590 @@
+- Have Header/Footer
+- **Header**
+  - Header have leading and trailing sections
+  - Leading section can have: 
+    - Icon
+    - Title
+    - Subtitle
+    - Counter
+  - Trailing section can have:
+    - Badge
+    - Text
+    - Link
+    - Icon Button
+- **Footer**
+  - Footer have leading and trailing sections:
+  - Leading section can have: 
+    - Title
+    - Subtitle
+  - Trailing section can have:
+    - Primary action button
+    - Secondary action button
+
+
+## Card API Constraints 
+
+**Header Leading Section**
+
+- `Title` & `Subtitle`
+  - Size & text variant cannot be changed
+- `Icon`
+  - Size & color cannot be changed
+- `Counter`
+  - Size cannot be changed
+  - Variant can be changed
+
+**Header Trailing Section**
+
+Only one of the component can be used at a time: 
+- Badge
+- Text
+- Link
+- IconButton - only tertiary variant
+
+Other guideline from design side: 
+
+- Never use a primary/secondary action in the header, footer are meant for such actions.
+  - This means it's not recommended to change variant of IconButton
+
+**Footer Leading Section**
+
+- `Title` & `Subtitle`
+  - Size & text variant cannot be changed
+
+**Footer Trailing Sections**
+
+- Only two action buttons can be present `primary` or `secondary`
+- Users can totally omit action buttons
+- They can also use either variants of buttons `primary` or `secondary`
+
+## API
+
+With the above constraints we can propose the Card API as follows: 
+
+Sample usage:
+
+```jsx
+import {
+  Card,
+  CardHeader,
+  CardHeaderLeading,
+  CardHeaderTrailing,
+  CardHeaderBadge,
+  CardHeaderIcon,
+  CardHeaderCounter,
+  CardBody,
+  CardFooter,
+  CardFooterLeading,
+  CardFooterTrailing,
+} from "@razorpay/blade"
+
+<Card surfaceLevel={2}>
+  <CardHeader>
+    <CardHeaderLeading  
+      title="Payments Links" 
+      subtitle="Share payment link via an email, SMS, messenger, chatbot etc." 
+      prefix={<CardHeaderIcon icon={DollarIcon} />}
+      suffix={<CardHeaderCounter amount={20} />}
+    />
+    <CardHeaderTrailing 
+      visual={<CardHeaderBadge variant="positive">NEW</CardHeaderBadge>} 
+    />
+  </CardHeader>
+  <CardBody>
+    Card Body Content
+  </CardBody>
+  <CardFooter>
+    <CardFooterLeading 
+      title="Card Footer Title" 
+      subtitle="Card footer subtitle" 
+    />
+    <CardFooterTrailing 
+      actions={{
+        primaryAction: { text: 'Know more', onClick: () => {} },
+        secondaryAction: { text: 'Read Docs', onClick: () => {} },
+      }}
+    />
+  </CardFooter>
+</Card>
+```
+
+
+### `Card` API
+
+
+| Prop           | Type   | Default | Description                                                                            | Required |
+| -------------- | ------ | ------- | -------------------------------------------------------------------------------------- | -------- |
+| `surfaceLevel` | `2, 3` | `2`     | Surface level of the card background color, use this based on where the card is placed |          |
```

Comment:
why can't we have cards on surface level 1? can you confirm with design once? usually the cards will start on top of surface level 1

Reply:
Yes saurav confirmed this earlier, cards will have surface level 2 and 3 only. Since most backgrounds where cards will be added will be starting from 1 only. So if the background is level1 card will be level2.